R/slice-chop.R

```diff
@@ -98,6 +98,7 @@ list_unchop <- function(x,
                         ptype = NULL,
                         name_spec = NULL,
                         name_repair = c("minimal", "unique", "check_unique", "universal"),
+                        error_arg = caller_arg(x),
```

Comment:
Since the error call is set to `current_env()`:

```suggestion
                        error_arg = "x",
```

Reply:
Oh good point!